packages/core/src/index.ts

```diff
@@ -67,6 +67,7 @@ export type BaseGraphQLServerOptions<TContext> = {
 
 /**
  * Base class that can be extended to create a GraphQL server with any HTTP server framework.
+ * @internal
```

Comment:
This is what we use in non-node env

Reply:
That is why I added a factor function for this: https://github.com/dotansimha/graphql-yoga/pull/741/files#diff-9a4ceebe7c6f86856371906c3f061d3b56b7457022b05179884a113e7ced67e8R185-R190

Our concern is that we don't want people other from us extend these classes and do funky stuff